### ISSUE TITLE
Error: Missing "./dist/sdk.esm.js" export in "@directus/sdk" package

### ISSUE THREAD

**Issue by cdwmhcc** [directus/directus#9544]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

## Nuxt.js 3

"@directus/sdk": "9.0.0" error:
 
``` javascript
[SSR] Error transforming ./node_modules/@directus/sdk/dist/sdk.esm.js: Error: Missing "./dist/sdk.esm.js" export in "@directus/sdk" package
```

"@directus/sdk": "9.0.0-rc.100" normal.

## Nuxt.js 3

"@directus/sdk": "9.0.0" error:
 
``` javascript
[SSR] Error transforming ./node_modules/@directus/sdk/dist/sdk.esm.js: Error: Missing "./dist/sdk.esm.js" export in "@directus/sdk" package
```

"@directus/sdk": "9.0.0-rc.100" normal.

### What version of Directus are you using?

v9.0.0

### What version of Node.js are you using?

16.12.0

### What database are you using?

mysql 8.0

### What browser are you using?

chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

running locally

**Comment by danilopolani** [comment]
Might be related to https://github.com/directus/directus/discussions/9500 (and https://github.com/directus/directus/issues/9541) and fixed by https://github.com/directus/directus/pull/9535 but not released yet

**Comment by cdwmhcc** [comment]
101 Still report the error

**Comment by joselcvarela** [comment]
Hello @cdwmhcc 
I just have installed `@directus/sdk@9.0.1` and I am not seeing those warnings.
But I am receiving this `XMLHttpRequest is not defined`.

Can you confirm with the latest version?

**Comment by cdwmhcc** [comment]
> Hello @cdwmhcc I just have installed `@directus/sdk@9.0.1` and I am not seeing those warnings. But I am receiving this `XMLHttpRequest is not defined`.
> 
> Can you confirm with the latest version?

`"@directus/sdk": "latest"`

![image](https://user-images.githubusercontent.com/8030134/141421614-a6346a81-c9b0-48d8-825e-78879c8b113a.png)

`Error transforming ./node_modules/@directus/sdk/dist/sdk.esm.js: Error: Missing "./dist/sdk.esm.js" export in "@directus/sdk" package`

But the strange thing is that I can find the `sdk.esm.js` file in the `\node_modules\@directus\sdk\dist\` directory.

SDK 9.0.0-rc.100 is working.I'm using Windows 10.

**Comment by rijkvanzanten** [comment]
This should be fixed in https://github.com/directus/directus/pull/10230

**Comment by cdwmhcc** [comment]
> This should be fixed in #10230

I used the source code of sdk typescript and it worked normally, haha.
